web/schemas/ID1A3.json

```diff
@@ -88,6 +96,27 @@
             "CHEXS_NSF"
         ]
     },
+    {
+        "key": "Affiliation",
+        "type": "list_str",
+        "optional": true,
+        "multiple": false,
+        "section": "User",
+        "description": "Select affiliation",
+        "placeholder": "AirForce",
+        "value": [
+            "",
```

Comment:
the first value will be used as default on in drop down menu on MetaData service web UI. Do you want it to be empty? Please confirm.

Reply:
The placeholder is not empty. It is set as "AirForce"

Reply:
ok, sorry I missed it